Gems/ONNX/Code/Source/Clients/Model.cpp

```diff
@@ -40,7 +57,7 @@ namespace ONNX
         }
 
         m_cudaEnable = initSettings.m_cudaEnable;
-        m_session = Ort::Session::Session(*m_env, initSettings.m_modelFile.c_str(), sessionOptions);
+        m_session = Ort::Session::Session(*m_env, modelFile.c_str(), sessionOptions);
```

Comment:
According to the documentation [Ort::Session::Session](https://onnxruntime.ai/docs/api/c/struct_ort_1_1_session.html) accept a `const char*`, so there is no need to have a wstring at all.
But if there was a one this is the only place where a conversion to wstring needs to take place.
```suggestion
        m_session = Ort::Session::Session(*m_env, AZStd::to_wstring(onnxGemRoot.Native()).c_str(), sessionOptions);
```

Reply:
Even though the docs say that a const char* is accepted, the function actually uses an internally defined ORTCHAR_T*, which on Win32 is defined as wchar_t. As such, it doesn't compile unless a const wchar_t* is passed in.